.env.development

```diff
@@ -4,6 +4,7 @@ REACT_APP_ENV=development
 REACT_APP_LOG_LEVEL=debug
 REACT_APP_SERVER_URL=https://good-server.herokuapp.com
 REACT_APP_GUN_PUBLIC_URL=https://goodgun-dev.herokuapp.com/gun
+REACT_APP_PUBLIC_URL=https://dev.gooddollar.org
```

Comment:
```suggestion
```

it's not need anymore. also, during development of web this var should be automatically resolved from the location.url